cmd/mcp.go

```diff
@@ -10,24 +10,70 @@ import (
 func NewMCPServerCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "mcp",
-		Short: "Start MCP server",
+		Short: "The Functions Model Context Protocol (MCP) server",
 		Long: `
 NAME
-	{{rootCmdUse}} mcp - start a Model Context Protocol (MCP) server
+	{{rootCmdUse}} mcp - The Functions Model Context Protocol (MCP) server
 
 SYNOPSIS
-	{{rootCmdUse}} mcp [flags]
+	{{rootCmdUse}} mcp [command] [flags]
 
 DESCRIPTION
-	Starts a Model Context Protocol (MCP) server over standard input/output (stdio) transport.
-	This implementation aims to support tools for deploying and creating serverless functions.
+	The Functions Model Context Protocol (MCP) server can be used to give agents
+	the power of Functions.
 	
-	Note: This command is still under development.
+	Configure your agentic client to use the MCP server with command
+	"{{rootCmdUse}} mcp start".  Then get the conversation started with
+
+	"Let's create a Function!".
+
+	IMPORTANT: This is an EXPERIMENTAL feature. The FUNC_ENABLE_MCP environment
```

Comment:
This is not "read only", right? 

perhaps add a warning on the "write aspect" ?

Reply:
I completely reworded the command help text to be both less abrasive and more informative.  Thanks!